### PR TITLE
Bunker Shield's item script fix.

### PR DESCRIPTION
**Pull request by crazystorm2017**
* **Addressed Issue(s)**: 
#3249 

* **Server Mode**: 
Renewal

* **Description of Pull Request**: 
Fix the item script for the Item ID: 28915 Bunker Shield.

### CODE REVIEW

File: db/re/item_db.txt [hunk 1]
```diff
@@ -11721,7 +11721,7 @@
...
 28910,Rectangular_Large_Sleeve,Rectangular Large Sleeve,4,20,,800,,80,,1,0xFFFFFFFF,63,2,32,,100,1,1,{ bonus bLongAtkDef,5; .@r = getrefine(); if(.@r>=5) { bonus bLongAtkDef,5; } if(.@r>=7) { bonus bLongAtkDef,10; } if(.@r>=9) { bonus bLongAtkDef,15; } },{},{}
 28913,Ultralight_Magic_Shield,Ultralight Magic Shield,4,20,,100,,50,,1,0xFFFFFFFF,63,2,32,,100,1,1,{ bonus bAspdRate,5; bonus2 bIgnoreDefClassRate,Class_Normal,10; bonus2 bSubEle,Ele_Neutral,5; .@r = getrefine(); if(.@r>=7) { bonus2 bIgnoreDefClassRate,Class_Normal,2; bonus bAspdRate,2; } if(.@r>=9) { bonus2 bIgnoreDefClassRate,Class_Normal,3; bonus bAspdRate,3; } },{},{}
-28915,Bunker_Shield,Bunker Shield,4,20,,3500,,90,,1,0xFFFFFFFF,63,2,32,,50,1,1,{ bonus2 bAddRace,RC_All,4; bonus bAspdRate,5; .@r = getrefine(); if(.@r>=5) { bonus2 bAddRace,RC_All,.@r; } },{},{}
+28915,Bunker_Shield,Bunker Shield,4,20,,3500,,90,,1,0xFFFFFFFF,63,2,32,,50,1,1,{ .@r = getrefine(); bonus2 bAddClass,Class_All,4; if(.@r >= 5){ bonus2 bAddClass,Class_all,.@r; } },{},{}
```
Comment (aleos89) [inline db/re/item_db.txt]: The ASPD bonus is gone?

**Comment by crazystorm2017** [comment]
@aleos89 missed it.

File: db/re/item_db.txt (hunk 1 above)
Comment (Atemo) [reply db/re/item_db.txt]: Actually the aspd should be `bonus bAspdRate,-5;` according to the description
https://www.divine-pride.net/database/item/28915/jRO
I would rather use the ternary operator `?` to concatenate the value of the class bonus

File: db/re/item_db.txt [hunk 2]
```diff
@@ -11721,7 +11721,7 @@
...
 28910,Rectangular_Large_Sleeve,Rectangular Large Sleeve,4,20,,800,,80,,1,0xFFFFFFFF,63,2,32,,100,1,1,{ bonus bLongAtkDef,5; .@r = getrefine(); if(.@r>=5) { bonus bLongAtkDef,5; } if(.@r>=7) { bonus bLongAtkDef,10; } if(.@r>=9) { bonus bLongAtkDef,15; } },{},{}
 28913,Ultralight_Magic_Shield,Ultralight Magic Shield,4,20,,100,,50,,1,0xFFFFFFFF,63,2,32,,100,1,1,{ bonus bAspdRate,5; bonus2 bIgnoreDefClassRate,Class_Normal,10; bonus2 bSubEle,Ele_Neutral,5; .@r = getrefine(); if(.@r>=7) { bonus2 bIgnoreDefClassRate,Class_Normal,2; bonus bAspdRate,2; } if(.@r>=9) { bonus2 bIgnoreDefClassRate,Class_Normal,3; bonus bAspdRate,3; } },{},{}
-28915,Bunker_Shield,Bunker Shield,4,20,,3500,,90,,1,0xFFFFFFFF,63,2,32,,50,1,1,{ bonus2 bAddRace,RC_All,4; bonus bAspdRate,5; .@r = getrefine(); if(.@r>=5) { bonus2 bAddRace,RC_All,.@r; } },{},{}
+28915,Bunker_Shield,Bunker Shield,4,20,,3500,,90,,1,0xFFFFFFFF,63,2,32,,50,1,1,{ bonus bAspdRate,-5; bonus2 bAddClass,Class_All,4; .@r = getrefine(); if(.@r >= 5){ bonus2 bAddClass,Class_all,.@r; } },{},{}
```
Comment (Atemo) [inline db/re/item_db.txt]: I noticed the bonus value is wrong, it should be 4+1% per refine after refine +4
`.@r = getrefine(); if(.@r >= 5) { .@dmg = .@r - 4; } bonus2 bAddClass,Class_All,(4+.@dmg);`

**Pull request merged**